inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```

Comment:
Avoid using confusing variable name. 
```suggestion
    mean = np.mean(data, axis=0)
```